website/models.py

```diff
@@ -312,6 +313,22 @@ def remove_user(self):
     def get_absolute_url(self):
         return "/issue/" + str(self.id)
 
+    def get_cve_score(self):
+        if self.cve_id:
```

Comment:
Consider checking `if self.cve_id is None` first:
```suggestion
        if self.cve_id is None:
            return None
```
to save an indent below